### DISCUSSION TITLE
Disabled because a matching PR was automerged previously.

### DISCUSSION THREAD

**Discussion by risenxxx** [renovatebot/renovate#23392] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

Renovate pull requests automerging in my repo stopped someday. This is pull request example:
https://github.com/eslint-kit/eslint-kit/pull/175

And the renovate configuration file:
https://github.com/eslint-kit/eslint-kit/blob/main/renovate.json

Renovate says: "Disabled because a matching PR was automerged previously". As I understand, the reason is grouped updates, causing the repeating branch names and PR titles. Also I don't understand why some Renovate PRs have only `chore(deps):` in title, but some for example `chore(deps): typescript preset`...

Maybe I'm doing something wrong and there is the way to allow this kind of PRs to automerge?

As I see, there are similar discussions, and there said that fix is done, but I'm still facing the issue:
https://github.com/renovatebot/renovate/discussions/22780
https://github.com/renovatebot/renovate/discussions/22770

### What you tried so far.

Almost nothing, because I don't see the way to fix the issue

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: Found PR #175 (branch="renovate/major-eslint-plugins")
DEBUG: branchExists=true (branch="renovate/major-eslint-plugins")
DEBUG: dependencyDashboardCheck=undefined (branch="renovate/major-eslint-plugins")
DEBUG: PR rebase requested=false (branch="renovate/major-eslint-plugins")
DEBUG: Check for closed PR because recreating closed PRs is disabled. (branch="renovate/major-eslint-plugins")
DEBUG: findPr(renovate/major-eslint-plugins, chore(deps):, !open) (branch="renovate/major-eslint-plugins")
DEBUG: Found PR #145 (branch="renovate/major-eslint-plugins")
DEBUG: Found closed PR with current title (branch="renovate/major-eslint-plugins")
DEBUG: Returning PR from cache (branch="renovate/major-eslint-plugins")
DEBUG: Matching PR #145 was merged previously (branch="renovate/major-eslint-plugins")
DEBUG: Disabling automerge because PR was merged previously (branch="renovate/major-eslint-plugins")
```

</details>

**Comment by rarkins** [comment]
Your commitMessageX configurations are causing a problem with the commit message and PR title

**Comment by MindTooth** [comment]
I see that you've combined Dependabot and Renovate.  Any reason for this?

I've created a [fork](https://github.com/MindTooth/eslint-kit/pulls) and a basic config in an attempt at duplicating yours:
```json
{
  "extends": [
    "config:recommended",
    ":pinDevDependencies",
    "group:linters",
    "schedule:automergeEarlyMondays"
  ],
  "packageRules": [
    {
      "extends": [":automergeAll", "packages:linters"]
    }
  ]
}
```

---

@rarkins off-topic: could e.g `rollup*` be a candiate for inclusion same is `eslint*`?   If so, I can submit a PR. https://github.com/renovatebot/renovate/blob/4b43966ca02bd10683cd4bcec3156a018d8f3469/lib/config/presets/internal/packages.ts#L23-L27

**Reply by risenxxx** [reply]
Dependabot is just for security updates

**Reply by MindTooth** [reply]
Renovate can act upon the same data I believe, and provide the same functionality[1].

[1]: https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts
